src/GraphQL.MicrosoftDI/GraphQLBuilder.cs

```diff
@@ -27,13 +27,15 @@ public class GraphQLBuilder : GraphQLBuilderBase, IServiceCollection
         /// Initializes a new instance for the specified service collection.
         /// </summary>
         /// <remarks>
-        /// Registers various default services via <see cref="GraphQLBuilderBase.Initialize"/>.
+        /// Registers various default services via <see cref="GraphQLBuilderBase.RegisterDefaultServices"/>
+        /// after executing the configuration delegate.
         /// </remarks>
-        public GraphQLBuilder(IServiceCollection services)
+        public GraphQLBuilder(IServiceCollection services, Action<IGraphQLBuilder>? configure)
         {
             Services = services ?? throw new ArgumentNullException(nameof(services));
             services.AddOptions();
```

Comment:
`services.AddOptions();` is a part of `RegisterDefaultServices` IMO.

Reply:
The options nuget package is not referenced by GraphQL.NET.  So it cannot be moved into GraphQLBuilderBase.RegisterDefaultServices.

I could override RegisterDefaultServices and move AddOptions in there.  But I thought that AddOptions should come before options configurations, not after.

Reply:
(It probably doesn't matter, however.)

Reply:
> I could override RegisterDefaultServices and move AddOptions in there.

👍

Reply:
Done